### PR TITLE
Typography Fixes

### PR DESCRIPTION
**Pull request by colophonemes**

### CODE REVIEW

File: packages/lesswrong/themes/eaForumTheme.js [hunk 1]
```diff
@@ -27,12 +33,23 @@ const palette = {
   }
 }
 
+const basicText = {
+  color: grey[700],
```
Comment (jpaddison3) [inline packages/lesswrong/themes/eaForumTheme.js]: I've decided this is too grey for post text. LW uses .87. I prefer that.

File: packages/lesswrong/themes/eaForumTheme.js [hunk 2]
```diff
@@ -4,11 +4,17 @@ import deepOrange from '@material-ui/core/colors/deepOrange';
 
 const sansSerifStack = [
   'Raleway',
+  'Helvetica Neue',
```
Comment (jpaddison3) [inline packages/lesswrong/themes/eaForumTheme.js]: On a regular post this kinda clashes with the text. Why is the title san serif anyway?

**Comment by jpaddison3** [comment]
image

**Comment by colophonemes** [comment]
^ Fixed on the `logo` branch

**Pull request merged**